package.json

```diff
@@ -36,7 +36,8 @@
     "handlebars": "2.0.0",
     "jit-grunt": "0.8.0",
     "jsonfile": "2.0.0",
-    "jspm": "0.14.0",
+    "jspm": "0.15.0",
+    "jspm-bower-endpoint": "^0.3.2",
```

Comment:
Dependencies brought down from bower: `socketio` and `videojspersistvolume`. The former can be fetched from npm (`socket.io-client` IIRC), the later can be fetched from GitHub. After discussions with @guybedford, it turns out the GitHub endpoint/registry does everything the Bower one does, including reading the Bower/package manifest for `main`.